depthai_ros_driver/include/depthai_ros_driver/pipeline.hpp

```diff
@@ -20,7 +22,7 @@ class Pipeline {
      * @brief The first function to get called, should have the setup ready for any future calls
      *
      */
-    void configure() { onConfigure(); }
+    void configure(const std::string& config_json = "") { onConfigure(config_json); }
```

Comment:
Thoughts on using YAML instead? It would offer better integration with ROS

Reply:
I followed depthai convention using json, but I don't think we need to stick to that convention because no json loading/reading/parameter feeding in gen2. I will add this to an TODO task.